pkgs/development/r-modules/default.nix

```diff
@@ -498,6 +498,7 @@ let
 
   packagesWithBuildInputs = {
     # sort -t '=' -k 2
```

Comment:
I don't know why this line is here it is not enforced before. @jbedo if you want to sort this list.